R/pkgdown.R

```diff
@@ -40,6 +42,23 @@ local_pkgdown_site <- function(path = NULL, meta = NULL, env = parent.frame()) {
     desc$write(file = path(path, "DESCRIPTION"))
     
     file_create(path(path, "_pkgdown.yml"))
+  } 
+
+  if (clone) {
```

Comment:
the argument name made me think a local Git clone would be involved

Reply:
Hmmm, yeah, `copy` would be a better name. But I'll probably end up refactoring this into a couple of different functions, so I'm going to leave it for now.